packages/flutter/lib/src/material/data_table.dart

```diff
@@ -414,21 +414,23 @@ class DataTable extends StatelessWidget {
     this.sortColumnIndex,
     this.sortAscending = true,
     this.onSelectAll,
-    this.dataRowHeight = kMinInteractiveDimension,
-    this.headingRowHeight = 56.0,
-    this.horizontalMargin = 24.0,
-    this.columnSpacing = 56.0,
+    this.decoration,
+    this.dataRowColor,
+    this.dataRowHeight,
+    this.dataTextStyle,
+    this.headingRowColor,
+    this.headingRowHeight,
+    this.headingTextStyle,
+    this.horizontalMargin,
+    this.columnSpacing,
     this.showCheckboxColumn = true,
+    this.showBottomBorder = false,
     this.dividerThickness = 1.0,
```

Comment:
Why isn't this part of the theme?

Reply:
Good point, I added dividerThickness to the theme now.